Homework_11/Task_1.py

```diff
@@ -0,0 +1,22 @@
+def celsius_to_fahrenheit(celsius: float):
+    return (celsius * 9/5) + 32
+
+def fahrenheit_to_celsius(fahrenheit: float):
+    return (fahrenheit - 32) * 5/9
+
+def convert_temperature(temperature: float, unit: str):
+    try:
+        if unit.upper() == 'C':
+            return celsius_to_fahrenheit(temperature)
+        elif unit.upper() == 'F':
+            return fahrenheit_to_celsius(temperature)
+        else:
+            raise ValueError("Error. Please use 'C' for Celsius or 'F' for Fahrenheit.")
+    except ValueError as e:
+        return str(e)
+
```

Comment:
if __name__ == '__main__':

Reply:
![image](https://github.com/senya2006/Semen-Homework/assets/51053483/2be4e942-023a-45bd-b484-86a8c8b51a0a)

Reply:
Also add in the last line of code {unit.upper()}